apps/events/tests/api_tests.py

```diff
@@ -1,426 +1,23 @@
-import json
-
 from django.contrib.auth.models import Group
 from django.urls import reverse
 from django.utils import timezone
 from django_dynamic_fixture import G
-from onlineweb4.fields.recaptcha import mock_validate_recaptcha
 from rest_framework import status
 
 from apps.authentication.models import OnlineUser
 from apps.companyprofile.models import Company
-from apps.events.constants import AttendStatus
-from apps.events.models import Attendee, CompanyEvent, GroupRestriction
+from apps.events.models import CompanyEvent, GroupRestriction
 from apps.online_oidc_provider.test import OIDCTestCase
 from apps.profiles.models import Privacy
 
-from .utils import (
-    attend_user_to_event,
-    generate_event,
-    generate_payment,
-    generate_user,
-    pay_for_event,
-)
+from .utils import attend_user_to_event, generate_event, generate_user
 
```

Comment:
Almost none of these tests are removed, hat have just been moved and fixed a bit.